### PR TITLE
Code restructuring

### PR DESCRIPTION
**Pull request by thak123**
DOnt merge

### CODE REVIEW

File: vitaflow/run/factory/dataset.py [hunk 1]
```diff
@@ -1,64 +1,72 @@
...
+            print('dataset.py: Found Config {}'.format(config))
+            with open(config) as fp:
+                try:
+                    _config_dict = load(fp)
+                    _datasets.update(_config_dict['dataset'])
+                    print('\t\t\t\t {}'.format(_config_dict['dataset']))
+                except FileNotFoundError:
+                    raise KeyError('Not able to find config or config key - dataset')
+    print('-----------------------------------' * 2)
+    return _datasets
```
Comment (thak123) [inline vitaflow/run/factory/dataset.py]: This and similar are not taken care while refactoring.

File: vitaflow/run/factory/dataset.py [hunk 2]
```diff
@@ -1,64 +1,72 @@
...
-    dataset_path = {
-        # file_name : package
-        "conll_2003_dataset": "vitaflow.data.text.conll.conll_2003_dataset",
-    }
-
-    datasets = {
-        # file_name : class_name
-        "conll_2003_dataset": "CoNLL2003Dataset"
-    }
+    @staticmethod
```
Comment (thak123) [inline vitaflow/run/factory/dataset.py]: This is not taken care while refactoring

File: vitaflow/run/factory/model.py [hunk 3]
```diff
@@ -1,69 +1,85 @@
...
 
+def search_n_load_models():
+    _models = dict()
+    print('-----------------------------------' * 2)
+    for config in data_model_configs:
+        print('model.py: Checking Config {}'.format(config))
+        with open(config) as fp:
+            try:
+                _config_dict = load(fp)
+                _models.update(_config_dict)
```
Comment (thak123) [inline vitaflow/run/factory/model.py]: Not taken care

**Comment by Mageswaran1989** [comment]
@thak123 closing this!

**Pull request closed without merging**